core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -2689,31 +2707,17 @@ public function overrideAll()
 
 				try
 				{
+					$this->preloadActiveRecords($ids);
+
 					foreach ($ids as $id)
 					{
-						try
-						{
-							$this->denyAccessUnlessGranted(ContaoCorePermissions::DC_ACTION_EDIT, new DataContainerSubject($this->strTable, $id));
-						}
-						catch (AccessDeniedException)
-						{
-							continue;
-						}
+						$this->loadActiveRecord($id);
 
-						$this->intId = $id;
```

Comment:
I don't think we should remove this, might be used by callbacks or event the `submit` method.

Reply:
The `loadActiveRecord()` does this.